lib/pliny/middleware/canonical_log_line.rb

```diff
@@ -0,0 +1,121 @@
+module Pliny::Middleware
+  # Emits a "canonical log line", i.e. a single log line that contains as much
+  # relevant information about a request as possible and which makes for a
+  # single convenient reference point to understand all the vitals of any
+  # single request.
+  #
+  # This default implementation contains some useful data to get a project
+  # started, but it's usually recommended to vendor this middleware into your
+  # project and start adding some custom fields. Some examples of those might
+  # be:
```

Comment:
Good question!

I definitely considered doing so, but decided against it (for now) for two reasons:

1. I think there's a pretty good likelihood that this isn't going to be the first thing newer projects want to start modifying. The default is probably going to serve you pretty well until you're in production and want to start building up some observability.
2. Because there's a chance that my "from scratch" implementation is not bug free, it would be nice to give this a little time to sit in master so that any bug fixes can be shipped directly as a Pliny patch.

I think given a little bit of time and maybe with some vetting at some non-Stripe companies, it would totally make sense to move this over to the template.